lib/stages/basebandReadout.cpp

```diff
@@ -128,8 +130,10 @@ void basebandReadout::main_thread() {
 
     std::unique_ptr<std::thread> wt;
     std::unique_ptr<std::thread> lt;
+    basebandReadoutManager *mgrs[_num_local_freq];
```

Comment:
The pointers in this array have to be set to `nullptr` (I don't think that is guaranteed automatically).